tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -56,7 +58,14 @@ auto Not(F &&f) -> decltype(ROOT::Internal::NotHelper(Args(), std::forward<F>(f)
    return ROOT::Internal::NotHelper(Args(), std::forward<F>(f));
 }
 
+template <typename Node>
+RInterface<ROOT::Detail::RDF::RNode> ToCommonNodeType(const RInterface<Node> &n)
```

Comment:
Might it be nicer to add a `RNode ToNode() const` function to the `RNode`? Or is that not the right thing to do / am I overlooking something?

Reply:
Users don't have a `RNode`, they have a `RInterface<RNode>`. `ToNode` could be a method of `RInterface<T>`, but I was hesitant to add methods to `RInterface` as the methods there as used in functional chains:
```c++
df.Filter(...).ToNode().Define(...);
```

Do we prefer this?

Reply:
I don't have strong opinions here. Another option:
```
class RNode: public RInterface<RNodeImpl> {
public:
  template <class IMPL>
  explicit RNode(const RInterface<IMPL>&);
```
such that people can do
```
RNode node{df.Filter()};
```

Reply:
uh, I think I like this best actually. It's still explicit, but it's just a constructor, so people know what to do with it, and there is no extra name to choose for the function that does the conversion.